src/www/ui/template/advice_license-edit.html.twig

```diff
@@ -12,6 +12,24 @@
   <input type="hidden" name="mod" value="advice_license"/>
   <input type="hidden" name="rf" value="{{ rf_pk }}"/>
   <table>
+  {% if rf_lastmodified is not empty %}
+    <tr>
+      <td>{{ 'Username Created'|trans }}:</td>
+      <td><span type="input" name="useridcreated"> {{ rf_user_fk_created|e }}</span></td>
```

Comment:
As per standard HTML, `<span>` tag does not have `type` and `name` attributes. Can you please either remove them or make it an `<input type="text">` element with `disabled` attribute?

Reply:
yeah okay